assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -365,7 +365,21 @@ function printView(url) {
 								#end 
 							</a>
 						</th>
-					#end	
+					#end
+					#if ($assignment.Estimate && !$assignment.Estimate.equals(""))
+						<th id="estimate">
+							<a href="javascript:void(0)" onclick="location='#toolLinkParam("AssignmentAction" "doSort_grade_submission" "criteria=$!sort_estimate")'; return false;"  title="$tlang.getString("gen.sorbyspenttime")">
+								$tlang.getString("gen.spenttime")
+								#if ($sortedBy.equals($!sort_estimate)) 
+									#if ($sortedAsc.equals("true")) 
+										<img id="sort_ascending_estimate" src = "#imageLink("sakai/sortascending.gif")" border="0" alt="$tlang.getString("gen.sorasc")" /> 
```

Comment:
How do these images look in dark theme? Is is possible to use font-awesome icons instead of images so it is easier to work in both themes?

Reply:
Done